tests/TESTFPC2.MLC

```diff
@@ -73,7 +76,8 @@ WTOPRC_END   EQU   *
          LA    R2,&VAL
          LA    R3,L'&VAL
          CALL  CVTTOHEX,((R2),DHEX,(R3))
-         WTO   MF=(E,WTOMSG)
+*        WTO   MF=(E,WTOMSG)                             #540
+         PUT   SYSOUT,WTOMSG                             #540
```

Comment:
Another WTO in immediately following IF .. ENDIF. Also, since the WTO is immediately followed by ABEND, should close the DCB.